### PR TITLE
feat(create-plugin): show prompt to choose Spanish support

### PR DESCRIPTION
**Pull request by tuanphamcybozu**
## Why
- We have started supporting Spanish (es) on plugin-manifest-validator recently.
- create-plugin has a prompt for language support, now it will support Spanish.

## What
- Add support Spanish in prompt
- Update readme
- Update manifest.json to display es field
- Update e2e test

## How to test
- `pnpm test:e2e`

## Checklist

- [x] Read [CONTRIBUTING.md](https://github.com/kintone/js-sdk/blob/master/CONTRIBUTING.md)
- [x] Updated documentation if it is required.
- [x] Added tests if it is required.
- [x] Passed `pnpm lint` and `pnpm test` on the root directory.

### CODE REVIEW

File: packages/create-plugin/src/messages.ts [hunk 1]
```diff
@@ -62,6 +62,26 @@ const messages = {
     en: "Plug-in description must be within 64chars",
     ja: "プラグインの中国語の説明を64文字以内で入力してください",
   },
+  Q_SupportEs: {
+    en: "Does your plug-in support Spanish?",
+    ja: "プラグインはスペイン語に対応していますか？",
```
Comment (tasshi-me) [inline packages/create-plugin/src/messages.ts:67]:
```suggestion
    ja: "スペイン語をサポートしますか？",
```

File: packages/create-plugin/src/messages.ts [hunk 2]
```diff
@@ -62,6 +62,26 @@ const messages = {
...
+  },
+  Q_NameEs: {
+    en: "Input your plug-in name in Spanish [1-64chars] (Optional)",
+    ja: "プラグインの名前を入力してください (スペイン語) [1-64文字] (省略可)",
```
Comment (tasshi-me) [inline packages/create-plugin/src/messages.ts:71]:
```suggestion
    ja: "プラグインのスペイン語名を入力してください [1-64文字] (省略可)",
```

File: packages/create-plugin/src/messages.ts [hunk 3]
```diff
@@ -62,6 +62,26 @@ const messages = {
...
+  },
+  Q_NameEsError: {
+    en: "Plug-in name must be within 64chars",
+    ja: "プラグイン名は64文字以内で入力してください",
+  },
+  Q_DescriptionEs: {
+    en: "Input your plug-in description in Spanish [1-200chars] (Optional)",
+    ja: "プラグインの説明を入力してください (スペイン語) [1-200文字] (省略可)",
```
Comment (tasshi-me) [inline packages/create-plugin/src/messages.ts:79]:
```suggestion
    ja: "プラグインのスペイン語の説明を入力してください [1-200文字] (省略可)",
```

File: packages/create-plugin/src/messages.ts [hunk 4]
```diff
@@ -62,6 +62,26 @@ const messages = {
...
+  },
+  Q_DescriptionEsError: {
+    en: "Plug-in description must be within 64chars",
+    ja: "プラグインの中国語の説明を64文字以内で入力してください",
```
Comment (tasshi-me) [inline packages/create-plugin/src/messages.ts:83]:
```suggestion
    ja: "プラグインのスペイン語の説明を200文字以内で入力してください",
```

File: packages/create-plugin/src/messages.ts [hunk 5]
```diff
@@ -62,6 +62,26 @@ const messages = {
     en: "Plug-in description must be within 64chars",
     ja: "プラグインの中国語の説明を64文字以内で入力してください",
```
Comment (tasshi-me) [inline packages/create-plugin/src/messages.ts:63]: The max letters of description seems wrong.
Can I ask you to fix it in this PR?

File: packages/create-plugin/src/messages.ts [hunk 6]
```diff
@@ -74,6 +94,10 @@ const messages = {
     en: "Input your home page url for Chinese (Optional)",
     ja: "プラグインの中国語のWebサイトURLを入力してください (省略可)",
   },
+  Q_WebsiteUrlEs: {
+    en: "Input your home page url for Spanish (Optional)",
+    ja: "スペイン語のホームページURLを入力してください (省略可)",
```
Comment (tasshi-me) [inline packages/create-plugin/src/messages.ts:99]:
```suggestion
    ja: "プラグインのスペイン語のWebサイトURLを入力してください (省略可)",
```

**Review by tasshi-me — CHANGES_REQUESTED** [review]
I added Japanese prompt messages!
Please apply it!

In addition, I found error messages for plugin description seems wrong, it can be within 200 letters but 64 letters in error messages.

**Review by hung-cybo — APPROVED** [review]
LGTM

**Review by tasshi-me — APPROVED** [review]

**Pull request merged**